### PR TITLE
Adding bls_sigs_app and pairing-plus-binding submodules.

### PR DESCRIPTION
**Pull request by nikhilvanjani**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -14,3 +14,9 @@
 	path = deps/cryptopp
 	url = https://github.com/weidai11/cryptopp
 	ignore = untracked
+[submodule "bls/pairing-plus-binding"]
+	path = bls/pairing-plus-binding
+	url = https://github.com/nikhilvanjani/pairing-plus-binding
+[submodule "bls/bls_sigs_app"]
+	path = bls/bls_sigs_app
+	url = https://github.com/nikhilvanjani/bls_sigs_app
```
Comment (ehildenb) [inline .gitmodules:22]: These should both be at `deps/pairing-plus-binding` and `deps/bls_sig_app`.

**Comment by ehildenb** [comment]
@nikhilvanjani can you provide some context about why these changes are needed?

Also, it would be really good to include a recipe for building these submodules in the Makefile here, and to run those build procedures on CI. That way we can be confident we know working formulae for building these pieces of code when we go to integrate them into other downstream projects.

File: .gitmodules (hunk 1 above)
Comment (nikhilvanjani) [reply .gitmodules:22]: That's what I thought in the first place. But Musab suggested to place them in a new directory bls.

**Comment by tothtamas28** [comment]
@ehildenb, can this be closed?

**Pull request closed without merging**